input/fsh/Ex-XRTS-03-Prostate-1P-3V.fsh

```diff
@@ -106,6 +110,10 @@ Usage: #example
   * extension[uniformFractionation].valueBoolean = true
   * extension[pointDose].valueBoolean = false
   * extension[primaryPlanDose].valueBoolean = false
+  * extension[biologicallyEffectiveDose].valueQuantity
+    * code = #BED(a/b=2.5)
```

Comment:
This expression should go to the unit string and then code and system would be the code for Gy and ucum system (not required by the profile). See also validation finding http://build.fhir.org/ig/HL7/codex-radiation-therapy/branches/bioeffdose/qa.html#_scratch_repo_fsh-generated_resources_SearchParameter-Patient-id

Reply:
The Patient-id issue is separate.
Fixed the code vs units.